imgr/imgrpkg/api.go

```diff
@@ -71,6 +74,12 @@
 //
 // This will cause the specified <volumeName> to no longer be served. Note that
 // this does not actually affect the contents of the associated Container.
+
+Does this mean that the "root" user on the system will be able to delete an active volume?
+Is that a security concern?
+
```

Comment:
Absolutely anyone with access to the embedded HTTP Server can wreak havoc... but, then, they can wreak havoc by directly addressing Swift as well of course. Need we be any more secure?

Reply:
Good point!    The root user could just do "rm -rf *" or the swift CLI

As long as we are no worse than that we should be fine.

Can you tell I have been reading too many security related PLC documents lately?

Reply:
> Can you tell I have been reading too many security related PLC documents lately?

:-)

One very valid concern that remains, however, is the ease with which one could do the POST API... all too easy to do that on a Container that has good data (whether or not it's ProxyFS data) in it alas... but I assume monitord will be the only one executing this "mkfs" command... and will do so knowing that it is "ok" to do...